docker/table-diff-handle/tablediffview/github_api_handle.py

```diff
@@ -0,0 +1,82 @@
+
+import re
+import requests
+from tablediffview.config import CONFIG
+from functools import reduce
+
+GITHUB_API_ISSUE_URL = (
+    f"{CONFIG.GITHUB_API_URL}/{CONFIG.GITHUB_REPO_OWNER}/{CONFIG.GITHUB_REPO_NAME}/issues"
+)
+
+ISSUE_NAME_REGEX = r'main|^release/\d*.\d*|^v\d*.\d*.\d*'
+REQUEST_HEADER = {
+    "Authorization": f"token {CONFIG.GITHUB_TOKEN}",
+    "Content-Type": "application/vnd.github+json"
+}
+@staticmethod
+def get_issue_number(branch_name: str):
+    extract_issue_name = re.findall(ISSUE_NAME_REGEX, branch_name)
+    issue_title = "{0} - Tables different"
+    if not extract_issue_name:
+        issue_title = issue_title.format(next(iter(extract_issue_name)))
+    else:
+        issue_title = issue_title.format(branch_name)
+    get_issues_response = requests.get(GITHUB_API_ISSUE_URL, headers=REQUEST_HEADER, json={"state": "open"})
+    if get_issues_response.status_code != 200:
+        raise SystemExit(get_issues_response.json())
+    open_issues = get_issues_response.json()
+    for issue in open_issues:
+        if issue["title"] == issue_title:
```

Comment:
@TruongQuangSB Why do you want always an issue. Can't we just post the comparison in the PR as a comment?

Reply:
In Github REST API dont give Comment-Request for PR. Issue here is a PR

Reply:
@TruongQuangSB Alright. Got it then